README.md

```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/Vermonster/node-fhir-client.svg?branch=master)](https://travis-ci.org/Vermonster/node-fhir-client) [![Coverage Status](https://coveralls.io/repos/github/Vermonster/node-fhir-client/badge.svg?branch=master)](https://coveralls.io/github/Vermonster/node-fhir-client?branch=master)
```

Comment:
Does the reference to `node-fhir-client` in the badges need updating as well?

Reply:
yes! I'll update it